cookiecutter-cogent3-plugin/hooks/post_gen_project.py

```diff
@@ -0,0 +1,58 @@
+import os
+from jinja2 import Template
+
+# Load the template content
+app_template_content = """
+{% raw %}
+from cogent3.app.composable import define_app
+
+@define_app()
+class {{ app_class_name }}:
+    def main(self, data: str) -> str:
```

Comment:
If you're going to define a class, may as well add a basic `__init_` method to the stub too. Also, can you add a commented out version that just uses a function intsead of the class.

Reply:
will do (commented out function, and __init__).

I have been running (many many times) the cookiecutter and then `pip install .`  the resulting folder and looking at the notebook and seeing the apps.  BTW almost have the notebook working as a template ... maybe another 30 mins I'll have that working too.

I _think_ it's creating the directory structure I intend ... but I may be missing something obvious.

Reply:
I'm going to add some sample docstrings too, so people know how to document apps